beast/tools/run/make_ast_inputs.py

```diff
@@ -11,12 +11,13 @@
 from beast.observationmodel.ast.make_ast_input_list import (
     pick_models,
     pick_models_toothpick_style,
+    supplement_ast,
 )
 from beast.observationmodel.ast import make_ast_xy_list
 from beast.tools import beast_settings
 
 
-def make_ast_inputs(beast_settings_info, flux_bin_method=True):
+def make_ast_inputs(pick_method,beast_settings_info):
```

Comment:
Given the docstring says that there is a default for pick_method, I was surprised this was not capture by `pick_method="flux_bin_method"` in the def line.  I can see how this works when this script is called, but if the `make_ast_inputs` is called as a function, then would the correct default be used?

Reply:
I changed 'pick_method' from a positional argument to a keyword argument with a default value of 'flux_bin_method'.